### ISSUE TITLE
FoxDot indent error

### ISSUE THREAD

**Issue by mathigatti** [munshkr/flok#36]
Hi, I get errors when I try to execute multiline commands in foxdot because of indentation issues.

It looks like something like this:
```
def a_function():
    p1 >> pluck()
```
is converted into this:
```
def a_function():
p1 >> pluck()
```

There might be some issue trim in the lines? Maybe [here](https://github.com/munshkr/flok/blob/master/packages/web/components/TextEditor.tsx#L90)?

**Comment by munshkr** [comment]
Yes, it's probably there! Actually this problem is also present on Tidal.

**Comment by munshkr** [comment]
Maybe it should trim only at the end of the line...

**Comment by munshkr** [comment]
Fixed on commits 94dab26a9622f9601d791df61f05ae32eff6eeb4 and e0580ac7875d9493e1b72bfa4ca28d690f0c59d8
